knora/dsplib/schemas/lists-only.json

```diff
@@ -1,5 +1,5 @@
 {
-    "$schema": "https://json-schema.org/draft-07/schema",
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
With `https`, the file is not found, which produces the following warning:
`DeprecationWarning: The metaschema specified by $schema was not found. Using the latest draft to validate, but this will raise an error in the future.`